frontend/src/services/translation.service.ts

```diff
@@ -1,21 +1,27 @@
 import { httpService } from './http.service';
-import { Segment } from '../types/frontend-types';
+import { Example } from '../types/frontend-types';
 
 const TRANSLATE = 'translate';
 
 
 export async function translateParagraphs(
-    paragraphs: string[],
-    source_language: string,
-    target_language: string
-  ): Promise<{
-    translated_paragraphs: string[];
-    properties: any;
-    total_segments_translated: number;
-  }> {
-    return await httpService.post(`${TRANSLATE}`, {
-      paragraphs,
-      source_language,
-      target_language
-    });
-  }
\ No newline at end of file
+  paragraphs: string[],
+  source_language: string,
+  target_language: string,
+  dictionary_id?: number,
+  dictionary_timestamp?: string,
+  examples?: Example[]
```

Comment:
We should decide at which point translate service is ignoring the database and only handling translations.
1. Create dictionary
2. Add prompt rule
3. Add example rules
4. Transform dictionary to prompt (as string)
5. Call translation service with final prompt.

In that way translation service does not have to know about our structure of dictionaries and rules and sources and segments....